cmd/start.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/weppos/publicsuffix-go/publicsuffix"
 )
 
+// in case of remote registry, an auto updater is set up
+const autoUpdatePeriod time.Duration = 12 * time.Hour
```

Comment:
the update period should be configurable with a flag instead of being hardcoded. Just a reminder.

Reply:
cc @louisinger